packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.8.4
+
+* Added `onScrubbingCallback` to `VideoProgressIndicator`
```

Comment:
https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates

Should start with `Adds` and end with `.`

Reply:
Adjusted